server/pbench/bin/pbench-satellite-cleanup

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+import sys
+import os
+for line in sys.stdin:
+    name = line.strip('\n')
+    try:
+        os.system('mv TO-SYNC/%s SYNCED' % name)
+    except Exception as e:
+        print (e)
```

Comment:
If this fails, we should not continue.   So lets print the exception, but use a "continue" statement to handle the next one.